CODEOWNERS

```diff
@@ -26,8 +26,24 @@
 /bgp/ @cilium/sig-bgp
 /cmd/ @cilium/cli
 /clustermesh/ @cilium/sig-clustermesh
-/connectivity/ @cilium/cli
+/connectivity/ @cilium/ci-structure
+/connectivity/tests/egressgateway.go @cilium/egress-gateway
+/connectivity/tests/encryption.go @cilium/sig-encryption
+/connectivity/tests/errors.go @cilium/sig-agent @cilium/sig-datapath
+/connectivity/tests/externalworkload.go @cilium/sig-clustermesh
+/connectivity/tests/from-cidr.go @cilium/sig-policy
+/connectivity/tests/health.go @cilium/sig-agent
+/connectivity/tests/host.go @cilium/sig-agent
+/connectivity/tests/ipsec_xfrm.go @cilium/ipsec
+/connectivity/tests/perfpod.go @cilium/sig-datapath
```

Comment:
Ideally this test would be owned by a team responsible for performance specific stuff. I don't think such a team exists (yet), maybe it's worth creating one?